apis/python/tests/ht/_array_state_machine.py

```diff
@@ -8,12 +8,13 @@
 import math
 import re
 from abc import abstractmethod
-from typing import Any, Literal, Protocol, Union
+from typing import Any, Literal, Protocol, Tuple, Union
```

Comment:
I noticed many other files use `tuple`, but this one uses `Tuple`.  Probably should pick one.

My understanding is that `tuple[...]` works fine in 3.9 onward, so I'd prefer that.